pkg/autoscaler/metrics/stats_scraper.go

```diff
@@ -204,10 +209,11 @@ func (s *serviceScraper) Scrape(window time.Duration) (stat Stat, err error) {
 		pkgmetrics.RecordBatch(s.statsCtx, scrapeTimeM.M(float64(scrapeTime.Milliseconds())))
 	}()
 
-	if s.podsAddressable {
+	if s.podsAddressable || s.usePassthroughLb {
```

Comment:
Is it possible that due to some istio/ingress reconciler problems this actually still fails and we never get to scrape? I.e. should be &&?

Reply:
Well with an `&&` we will not scrape if passthrough LB is disabled. We also don't want to fallback to service scraping ever. The goal is to eventually make this the default and then remove all the fallback code, so it either works always or we have a bug I'd say.

Reply:
OK :)